webapp/components/profile_popover/profile_popover.jsx

```diff
@@ -243,16 +260,31 @@ export default class ProfilePopover extends React.Component {
                     </a>
                 </div>
             );
-            dataContent.push(webrtc);
         }
 
+        return '';
+    }
+
+    render() {
+        const popoverProps = Object.assign({}, this.props);
```

Comment:
These props aren't actually used by the popover, so you can get rid of `popoverProps` entirely